### DISCUSSION TITLE
Unable to use reactQuery from a package.

### DISCUSSION THREAD

**Discussion by osvald0** [vercel/turborepo#8561] [Help] [answered]
### Summary

I have a `/hooks` package where I use React Query. I have the React Query Providers in the apps, but They are not working. Please check the error here:

![Screenshot 2024-06-20 at 7 58 17 PM](https://github.com/vercel/turbo/assets/20977158/c08a7b6f-0c16-453d-8110-db835a976939)

The React Query Providers are defined as expected in the root of the projects.
What is the right way to make this working?

**Comment by osvaldo-cv** [comment]
Any comment?

**Reply by jimmy-ventura-elasticpath** [reply, marked as answer]
check this one https://github.com/TanStack/query/issues/3595
